js/src/product-feed/submission-success-guide/enhanced-conversions/index.js

```diff
@@ -0,0 +1,181 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment, createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import GuidePageContent from '.~/components/guide-page-content';
+import TrackableLink from '.~/components/trackable-link';
+import useAcceptedCustomerDataTerms from '.~/hooks/useAcceptedCustomerDataTerms';
+import useGoogleAdsEnhancedConversionSettingsURL from '.~/hooks/useGoogleAdsEnhancedConversionSettingsURL';
+import AppSpinner from '.~/components/app-spinner';
+
+const EnhancedConversions = () => {
+	const { acceptedCustomerDataTerms, hasFinishedResolution } =
+		useAcceptedCustomerDataTerms();
+	const url = useGoogleAdsEnhancedConversionSettingsURL();
+
+	let title = __(
+		'Optimize your conversion tracking with Enhanced Conversions',
+		'google-listings-and-ads'
+	);
+
+	if ( acceptedCustomerDataTerms ) {
+		title = __(
+			'Your Enhanced Conversions are almost ready',
+			'google-listings-and-ads'
+		);
+	}
+
+	const getPageContentBody = () => {
+		if ( ! hasFinishedResolution ) {
+			return <AppSpinner />;
+		}
+
+		if ( ! acceptedCustomerDataTerms ) {
```

Comment:
The copy for this has been updated so that the panel is not as long. See: https://docs.google.com/document/d/1U1Wiymz51aRsUZQKEHit-y9o1kiSdkEcBV-8e3BXJiM/edit#heading=h.27ah3dar1ze0.

Reply:
Updated @joemcgill